File: test2.py
import time
start_time = time.time()
n = 1_000_000_000
print("Массив был создан")
arr = [0] * n
print("Начал заполнять массив: ", time.time() - start_time)
for i in range(n):
    arr[i] = i
print("Начал считать сумму: ",  time.time() - start_time)
total = 0
for i in range(n):
    total += arr[i]

end_time = time.time()

print("")
print("Количество элементов: ", n)
print("Сумма: ", total)
print("Время работы: ", end_time - start_time)
# By ArisNow0

File: test2.cs
using System;
using System.Diagnostics;

class Program
{
    static void Main()
    {
    var stopwatch = Stopwatch.StartNew();

    long n = 1_000_000_000; // Max value(fuck c#)
    var arr = new int[n];
    for (int i = 0; i < n; ++i) {
        arr[i] = i;
    }
    long total = 0;
    for (int i = 0; i < n; ++i) {
        total += arr[i];
    }

    stopwatch.Stop();

    Console.WriteLine("Количество элементов: " + n);
    Console.WriteLine("Сумма: " + total);
    Console.WriteLine("Время: " + stopwatch.Elapsed.TotalSeconds + " секунд");
    }
} // By ArisNow0